workers/tasks/export/styles/printDocument.scss

```diff
@@ -23,206 +23,206 @@ https://gitlab.pagedmedia.org/tools/pagedjs-cli/issues/11 */
-        @top-left {
-            color: #666;
-            font-size: 10px;
-            font-family: $header-font;
-            content: string(community-and-collection);
-        }
-        @top-right {
-            color: #666;
-            font-size: 10px;
-            font-family: $header-font;
-            content: string(title);
-        }
-        @bottom-center {
-            color: #666;
-            font-size: 10px;
-            font-family: $header-font;
-            content: counter(page);
-        }
-    }
-    /* Avoid being the last element on the page */
-    h1,
-    h2,
-    h3,
-    h4,
-    h5,
-    h6 {
-        break-after: avoid;
-    }
-
-    tr,
-    th {
-        break-inside: avoid;
-        max-height: 90vh;
-        overflow-y: hidden;
-    }
-
-    figure {
-        break-inside: avoid;
-        display: flex;
-        flex-direction: column;
-        max-height: 800px;
-        img,
-        video {
-            break-inside: avoid;
-            width: 100%;
-            min-height: 0;
-            max-height: 100%;
-            flex-shrink: 1;
-            object-fit: contain;
-        }
-    }
-
-    figcaption {
-        font-size: 10px;
-        color: #555;
-        flex-shrink: 0;
-    }
-
-    table {
-        font-size: 12px;
-        border-collapse: collapse;
-        table-layout: fixed;
-        width: 100%;
-        overflow: hidden;
-    }
-
-    table caption {
-        text-align: left;
-    }
-
-    table,
-    tr,
-    th,
-    td {
-        border: 1px #ccc solid;
-        padding: 5px;
-    }
-
-    th {
-        font-weight: bold;
-        text-align: left;
-        background-color: #f0f0f4;
-    }
-
-    td,
-    a {
-        word-break: break-word;
-    }
-
-    a.footnote {
-        vertical-align: super;
-        font-size: 10px;
-    }
-
-    [data-node-type='math-block'] {
-        break-inside: avoid;
-    }
-
-    section.cover {
-        page: cover;
-    }
-
-    @page cover {
-        @top-left {
-            content: '';
-        }
-        @top-right {
-            content: '';
-        }
-        @bottom-center {
-            content: '';
-        }
-    }
-
-    .community-and-collection {
-        string-set: community-and-collection content();
-    }
-
-    .title {
-        string-set: title content();
-    }
+	/* Baseline page styles */
+	@page {
+		size: Letter;
+		@top-left {
+			color: #666;
+			font-size: 10px;
+			font-family: $header-font;
+			content: string(community-and-collection);
+		}
+		@top-right {
+			color: #666;
+			font-size: 10px;
+			font-family: $header-font;
+			content: string(title);
+		}
+		@bottom-center {
+			color: #666;
+			font-size: 10px;
+			font-family: $header-font;
+			content: counter(page);
+		}
+	}
+	/* Avoid being the last element on the page */
+	h1,
+	h2,
+	h3,
+	h4,
+	h5,
+	h6 {
+		break-after: avoid;
+	}
+
+	tr,
+	th {
+		break-inside: avoid;
+		max-height: 90vh;
+		overflow-y: hidden;
+	}
+
+	.editor figure {
```

Comment:
this is the only actual line changed. not sure why it did this to the entire file